src/onegov/event/upgrade.py

```diff
@@ -55,3 +57,12 @@ def add_filter_keywords_column(context):
             context.operations.add_column(
                 table, Column(column_name, MutableDict.as_mutable(
                     HSTORE), nullable=True))
+
+
+@upgrade_task('Adds default value for event filter type')
+def add_default_setting_for_event_filter_type(context):
+    org = context.session.query(Organisation).first()
+
+    if org and 'event_filter_type' not in org.meta:
+        org.meta['event_filter_type'] = 'tags'
+
```

Comment:
I don't think this is needed since you provided a default value on the `meta_property`, itwill return its default value if it hasn't been set yet.

Reply:
ok, got you